go.sum

```diff
@@ -112,6 +114,8 @@ github.com/envoyproxy/go-control-plane v0.9.0/go.mod h1:YTl/9mNaCwkRvm6d1a2C3ymF
 github.com/envoyproxy/go-control-plane v0.9.1-0.20191026205805-5f8ba28d4473/go.mod h1:YTl/9mNaCwkRvm6d1a2C3ymFceY/DCBVvsKhRF0iEA4=
 github.com/envoyproxy/go-control-plane v0.9.4/go.mod h1:6rpuAdCZL397s3pYoYcLgu1mIlRU8Am5FuJP05cCM98=
 github.com/envoyproxy/protoc-gen-validate v0.1.0/go.mod h1:iSmxcyjqTsJpI2R4NaDN7+kN2VEUnK/pcBlmesArF7c=
+github.com/ezamriy/gorpm v0.0.0-20160905202458-25f7273cbf51 h1:gyBeYaqi18UiHg0qrDN5nHxv5fNCjyLknzxLGTifRBU=
+github.com/ezamriy/gorpm v0.0.0-20160905202458-25f7273cbf51/go.mod h1:ALNC6ia2u6HaxQZFK2b9kdBCD2PyYmqToaO1kL61/Mw=
```

Comment:
Do we really need both MichaelMraka and ezamriy versions of this gorpm lib?

Reply:
I'd prefer to use upstream of the package but it has a bug which make is impossible to build.

Reply:
We can use just my fork but we won't get other upstream fixes in future.

Reply:
I think it's ok we use alias in go.mod. I just wonder we still need to have both repos in go.sum. I would guess we need just your fork there as this is really used under upstream alias.

Reply:
I see, I'll remove it.